closed/autoconf/custom-hook.m4

```diff
@@ -37,7 +37,7 @@ AC_DEFUN_ONCE([CUSTOM_EARLY_HOOK],
   AC_SUBST(CONFIG_SHELL)
 
   OPENJ9_BASIC_SETUP_FUNDAMENTAL_TOOLS
-  OPENJ9_PLATFORM_SETUP
```

Comment:
This is required by `OPENJ9_CONFIGURE_CMAKE`. The `OPENJ9_CONFIGURE_*` could be sorted but not others, otherwise the configure fails
```
16:16:42  checking which variants of the JVM to build... server
16:16:42  checking for m4... /usr/bin/m4
16:16:42  configure: error: --with-mixedrefs=[static|dynamic] requires --with-cmake
16:16:42  configure exiting with result code 1
```